### DISCUSSION TITLE
A Busylight for MS Teams with Tasmota LED Devices

### DISCUSSION THREAD

**Discussion by Claus-E** [arendst/Tasmota#10879] [Show and tell]
Hi there,

I would like to draw your attention to my first publication in connection with Tasmota. At https://github.com/claus-e/teams2tasmota I have published a software that can control a Tasmota device as a busylight for MS teams. I would be happy if someone can use it.
Greetings Claus

**Comment by barbudor** [comment]
Thanks for sharing 👍

**Reply by meingraham** [reply]
@Claus-E 

Thanks for sharing. I just purchased a Kuando BusyLight last month! Would have loved to try this. But will definitely file this in the event that ever stops working.

**Reply by Claus-E** [reply]
I would have bought the Kuando BusyLight too, but the USB cable bothered me :-) I wanted the Busylight to be in front of the door.
Claus

**Comment by Jason2866** [comment]
Very usefull :-)

**Comment by fruijs** [comment]
Thanks, it works great!! Thank you for sharing.

**Reply by Claus-E** [reply]
Thank you for the positive feedback. Let me know if anything doesn't go well.
Greetings Claus

**Reply by Triplex66** [reply]
Hello Claus, thanks for you Projekt to Share.  Do you think it is Possibly you can Signal a Incomming Call with flash LED or only Staple Collor?
Greetings from Hamburg
Thomas

**Reply by Claus-E** [reply]
Hi Thomas,

until now I don't have the need for this. After your question I looked at the log file again. And yes it looks like you can find the start of ringing too.

So please write a feature request on https://github.com/Claus-E/teams2tasmota/discussions/categories/ideas so other users can find it in the context of teams2tasmota. If I can find a few hours, I will try to implement it.

Greetings from a village near Hildesheim.
Claus
